packages/terra-i18n/scripts/aggregate-translations/aggregate-messages.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const fs = require('fs');
+
+const aggregateTranslationMessages = (translationDirectories, locales) => {
+  const translations = {};
+  locales.forEach((language) => {
```

Comment:
You don't have to but you could make this: const translations = locales.map(locale => {});

Reply:
I think its' reduce that you want here actually, as translations is an associated array.